markitdown_playground/apify_proxy_ip.py

```
@@ -0,0 +1,34 @@
+
+        # プロキシURLを取得
+        proxy_url = await proxy_configuration.new_url()
+
+        # requests用プロキシ設定
+        proxies = {
+            "http": proxy_url,
+            "https": proxy_url,
+        }
+
+        # テストリクエスト
+        response = requests.get(
```

Ruff (0.16.2)

[warning] Async functions should not call blocking HTTP methods

(ASYNC210)